### ISSUE TITLE
“No profile” Error After Connecting Microsoft Account

### ISSUE THREAD

**Issue by therealgreenrocket** [MultiMC/Launcher#5120]
### Description of bug

I had previously gotten this to work, but I have just reinstalled MultiMC, and now connecting my Minecraft account profile to MultiMC isn't working. Logging in and being redirected to the Microsoft website all works, but then the profile loaded into MultiMC says "No profile (my username)", with the default grey logo. I'm not able to launch Minecraft now out of MultiMC because there is no working connected profile it seems. Also yes, this is my same Microsoft account and profile that I DO play Minecraft on in the Minecraft launcher, so this account does have the game on it. 

Anyone know how to fix this? Thank you!

https://imgur.com/a/mq8gOif

### Steps to reproduce

Removing connected Microsoft account and reconnecting the account to MultiMC still maintains this issue.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by phit** [comment]
looks like you logged into an account without Minecraft profile, or it doesn't own the game, log in at minecraft.net the name in () is your xbox username

**Comment by therealgreenrocket** [comment]
I did that and it is the same exact email address and username connected to my account which I play Minecraft on in the Minecraft launcher. Here is a support request I sent to Microsoft about the issue.

"This leads into my second issue which is originally where this became a problem for me. I am using a popular application, MultiMC, to load Minecraft mods into my game. I have used it before with this account, but I recently reinstalled it, and now when I sign into MultiMC with my Minecraft Microsoft account, and after I get redirected to the Microsoft page to connect my account, it says I again do not own the game on MultiMC when I am logged in and it won't let me boot up the game. I am not sure why this is the case, as I have previously used the same exact account on MultiMC. I have contacted MultiMC support and asked on several places, all of which seem to work for everyone else right now. But I have just been told it seems I do not have a Minecraft purchase of the game on my Microsoft account, which is why I am contacting support to figure out why this is the case on my Microsoft account, even though I can still play the game already once I am on my launcher."
